### DISCUSSION TITLE
cy-12r ride

### DISCUSSION THREAD

**Discussion by ocimjr** [corrados/edrumulus#87] [General]
I see in the code that you implemented a basic support for a 3 cy-12r.

Since I have, I would like to tested. But I'm not sure about Wich pin I should use with prototype 2.

Thanks in advance,

**Comment by corrados** [comment]
The way it is implemented is that you select a pad which is coupled with your CY-12R pad. For my tests I have used the "tom2" pad. But you could use any pad for the coupling. Make sure to select the type CY (e.g. CY6):
![grafik](https://github.com/corrados/edrumulus/assets/46655886/72fcb65f-de6f-40bf-822c-19b3751d0d1b)
And then the coupling with, e.g., "tom2" pad which is index 7:
![grafik](https://github.com/corrados/edrumulus/assets/46655886/b0c43b31-0a1a-4c5f-878d-60e2714dac5a)

**Reply by corrados** [reply]
BTW, have you managed to run edrumulus_gui.py for configuring Edrumulus yet?

**Reply by ocimjr** [reply]
thanks. Yeah the gui is working no problem, but I didn't understand the couple pad question :)

**Reply by corrados** [reply]
The CY-12R has two jack inputs. So, you need to have two outputs in Edrumulus. Therefore, you have to couple your Ride channel with another one.

**Reply by ocimjr** [reply]
I was testing and didn't have good results, so :
1. Do you use 4 inputs, 2 stereo jacks, for this ?
2. Let's suppose input 6 is coupled with 7,  do you do note configuration on input 6, with note2 values ?
3. If we have 51(bow), 53(bell) and 59 (edge), how do you configure notes?
4. I didn't have choke, is it normal?

Thanks again

**Reply by corrados** [reply]
1. yes, I do, but it is not necessary. You only need three inputs, i.e., one stereo and one mono input.
2. yes, all MIDI assignments are configured in channel 6.
3. As above, use the MIDI assignments in channel 6 (you only need three of the four available settings).
4. You should have choke.

**Comment by ocimjr** [comment]
Thanks for your answer. I tested briefly today and it's working. I didn't have the time dial in the settings but seems good to me.
